salt/modules/inspectlib/kiwiproc.py

```diff
@@ -82,7 +82,7 @@ def export(self, name):
             [
                 line
                 for line in minidom.parseString(
-                    etree.tostring(root, encoding="UTF-8", pretty_print=True)
```

Comment:
Do we need to drop this also here? This is 3002.2 and should be only for python3

Reply:
there is no pretty_print keyword parameter there and it's actually converting to pretty in the next line anyway. It was dropped by upstream and already not present in 3004

Reply:
ACK